Adding snapshot options documentation (#34224)
dotnet/docs: docs/core/additional-tools/dotnet-coverage.md

  Sets the code coverage report output file. If not provided, it's generated automatically with a timestamp.

* **`-tn|--tag-name <tag-name>`**

>   Creates a snapshot tag name in the coverage file with current coverage information. Tag-name and tag-identifier should be provided together.

Reviewer: ```suggestion
  Creates a snapshot tag name in the coverage file with current coverage information. Tag-name and tag-identifier are mutually inclusive.
```
